saa/1.1. Interacting with services - AWS CLI & SDK.md

```diff
@@ -34,19 +34,20 @@
       - If EC2 is shared -> other people may perform actions impersonating you
   - 💡 Better way: AWS IAM Roles
     - IAM Roles can be attached to EC2 instances.
-      - ❗ One EC2 instance can have one IAM role at a time
+      - ❗ One EC2 instance can have one IAM role at a time(but the same role can be attached to many other EC2 instances)
```

Comment:
nice! can we have whitespace before the parenthesis or remove the parenthesis completely?

Reply:
Sure @undergroundwires we can have the blank space before the parenthesis.